src/sentry/services/hybrid_cloud/rpc.py

```diff
@@ -350,6 +373,20 @@ def create_delegation(cls, use_test_client: bool | None = None) -> Self:
         return service  # type: ignore[return-value]
 
 
+def list_all_service_method_signatures() -> Iterable[RpcMethodSignature]:
+    """List signatures of all RPC methods in the global registry."""
+
+    from .. import hybrid_cloud as hybrid_cloud_service_pkg
+
+    # Forcibly import all service packages to ensure the global registry is fully populated
+    for (importer, name, is_pkg) in pkgutil.iter_modules(hybrid_cloud_service_pkg.__path__):
+        if is_pkg:
+            importer.find_module(name).load_module(name)
```

Comment:
`is_pkg` is only going to get `__init__.py` files so this is not what you want

the recipe I usually use is:

```python
    for _, name, _ in pkgutil.walk_packages(hybrid_cloud_service_pkg.__path__, f'{ybrid_cloud_service_pkg.__name__}.'):
        __import__(name, fromlist=['_trash'])
```